src/abaqus/Feature/Feature.py

```diff
@@ -2,8 +2,8 @@
 
 from typing import Sequence, Union, overload
 
+from numpy.typing import NDArray
 from typing_extensions import Literal
-
```

Comment:
Please revert the deletion of the blank line because I want the changes to be clean and only necessary changes can be made.